turf/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,96 @@
+## Reference
+
+### TURF
+
+For more detailed reference on turf visit https://turfjs.org/docs/.
+#### Examples:
+
+```
+ST_BUFFER(geometry_to_buffer GEOGRAPHY, radius NUMERIC, units STRING, steps NUMERIC)
+```
+As taken from https://turfjs.org/docs/#buffer
+
+``` sql
+SELECT jslibs.turf.ST_BUFFER(ST_GEOGPOINT(-74.00,40.7128),1,'kilometers',10) as geo
+```
+
+#### VERSION
+
+{{% bannerNote type="code" %}}
+turf.VERSION()
+{{%/ bannerNote %}}
+
+Returns the current version of the turf library.
+
+#### BBOX
+
+{{% bannerNote type="code" %}}
```

Comment:
Why are we exposing this? Why are we asking users to do turf.BBOX(ST_AsGeoJSON(g)) instead of having a ST_BBOX or similar. Also, how is this different from `tiler.__WKB_ENVELOPE`?